client/app/components/composites/SideWinder/SideWinder.css

```diff
@@ -0,0 +1,78 @@
+.wrapper {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  right: 0;
+  transition: right 0.5s;
+}
+
+.winderOpen {
+  overflow-y: hidden;
```

Comment:
@rap1ds No, that is `overflow-y` on purpose. It prevents being able to scroll the page in the background vertically when the overlay is open.